packages/editor/src/store/actions.js

```diff
@@ -493,12 +493,9 @@ export function* trashPost() {
 			}
 		);
 
-		// TODO: This should be an updatePost action (updating subsets of post
-		// properties), but right now editPost is tied with change detection.
 		yield dispatch(
 			STORE_KEY,
-			'resetPost',
-			{ ...post, status: 'trash' }
+			'savePost'
```

Comment:
What does it mean to save a deleted post? My first impression is that we're abusing action creators a little in order to get a result not intended from them.

My second note is that the outbound comment mentions that `editPost` isn't used because of its ties to change detection, while `savePost` itself dispatches `editPost`.

Reply:
>What does it mean to save a deleted post? My first impression is that we're abusing action creators a little in order to get a result not intended from them.

It saves and refetches the post, getting the new status.

>My second note is that the outbound comment mentions that editPost isn't used because of its ties to change detection, while savePost itself dispatches editPost.

I can't think of any issues it could cause.